### ISSUE TITLE
Planning an API?

### ISSUE THREAD

**Issue by nbgoodall** [deepfates/memery#1]
This is phenomenal, thank you for building it. 🙏 

I noticed in the todos you've got "Distributed server backend" — are you planning to create an API for this?

**Comment by deepfates** [comment]
Yes! Ideally, I'd like to be able to run a server that can track databases by user. So there could be Android/iPhone clients etc. without having to run giant PyTorch models on the device. 

I don't really know how to to do this yet though, so any thoughts would be welcome. Have you got the repo working locally?
